app/layout/aside_feed.phtml

```diff
@@ -109,6 +109,9 @@
 	</form>
 </div>
 
+<div id="first_load" class="loading"></div>
```

Comment:
Moved as high as possible

Reply:
Can't we move these two lines between `$this->partial('aside_feed');` and `$this->partial('nav_menu');` (in normal and reader files?) It looks unnatural to have them in this file (i.e. they are not part of the aside), and so it's more difficult to understand their purpose.

Reply:
The best would be to change the CSS rule for `div#aside_feed` to ensure that it does not take the full width when it is the sole element while loading (a consequence of  `display:table-cell`), which is responsible for a layout flash in particular in Chrome. This `#first_load` is thus a helper for its previous sibling `div#aside_feed`.
Moving it between `$this->partial('aside_feed');` and `$this->partial('nav_menu');` would have two small drawbacks:
* It would need to be repeated for `normal.phtml` and for `reader.phtml`
* It would appear ~1.6KB later which, depending on where the buffering cuts, reduces a bit its chances of being visible

I suggest to leave it as it is and come back to it next time we optimise the CSS layout

Reply:
Ok then :)